packages/lesswrong/server/vulcan-lib/apollo-server/context.ts

```diff
@@ -94,12 +94,20 @@ export const generateDataLoaders = (): {
 };
 
 
-export function requestIsFromGreaterWrong(req?: Request): boolean {
+export function requestIsFromUserAgent(req?: Request, userAgentPrefix: string): boolean {
```

Comment:
Should probably accept a non-optional `req` and `requestIsFromGreaterWrong` and `requestIsFromIssaRiceReader` should just return false if req is undefined